job_test.go

```diff
@@ -492,3 +492,39 @@ func TestWithEventListeners(t *testing.T) {
 		})
 	}
 }
+
+func TestJob_NextRun(t *testing.T) {
+	testTime := time.Now()
+
+	s := newTestScheduler(t)
+
+	// run a job every 10 milliseconds that starts 10 milliseconds after the current time
+	j, err := s.NewJob(
+		DurationJob(
+			100*time.Millisecond,
+		),
+		NewTask(
+			func() {},
```

Comment:
```suggestion
			func() { time.Sleep(3 * time.Second) },
```

Could you also add some tests where this sleeps through _several_ of the next jobs that would run.